[11:48:44] [DEBUG] Running client sanity checks...
[11:48:44] [DEBUG] Loaded Extension: interactions.ext.jurigged
[11:48:44] [DEBUG] Added callback: __join
[11:48:44] [DEBUG] Added callback: __leave
[11:48:44] [DEBUG] Added callback: _about
[11:48:44] [DEBUG] Added callback: _setup
[11:48:44] [DEBUG] Added callback: on_start
[11:48:44] [DEBUG] 5 callbacks have been loaded from __main__.
[11:48:44] [DEBUG] Added callback: _disconnect
[11:48:44] [DEBUG] Added callback: _on_websocket_ready
[11:48:44] [DEBUG] Added callback: on_autocomplete_completion
[11:48:44] [DEBUG] Added callback: on_autocomplete_error
[11:48:44] [DEBUG] Added callback: on_command_completion
[11:48:44] [DEBUG] Added callback: on_command_error
[11:48:44] [DEBUG] Added callback: on_component_completion
[11:48:44] [DEBUG] Added callback: on_component_error
[11:48:44] [DEBUG] Added callback: on_error
[11:48:44] [DEBUG] Added callback: on_modal_completion
[11:48:44] [DEBUG] Added callback: on_modal_error
[11:48:44] [DEBUG] Added callback: on_resume
[11:48:44] [DEBUG] 12 callbacks have been loaded from Client.
[11:48:44] [DEBUG] Attempting to login
[11:48:45] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[11:48:45] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[11:48:45] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[11:48:45] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[11:48:45] [DEBUG] GET /oauth2/applications/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[11:48:45] [DEBUG] GET /oauth2/applications/@me Received 200 :: [0/1000 calls remaining]
[11:48:45] [WARNING] Setting sync_ext to True by default for syncing changes
[11:48:45] [INFO] Loading jurigged
[11:48:46] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\Main.py
[11:48:46] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[11:48:46] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[11:48:46] [INFO] Shard ID 0 | Starting Shard
[11:48:46] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[11:48:46] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[11:48:46] [INFO] Shard ID 0 | Gateway connection established
[11:48:46] [DEBUG] Shard ID 0 | Session ID: bcbe05c79a3b3d3783a94af81d9b33e6 Trace: ['["gateway-prd-us-east1-b-zv54",{"micros":68323,"calls":["id_created",{"micros":747,"calls":[]},"session_lookup_time",{"micros":11258,"calls":[]},"session_lookup_finished",{"micros":14,"calls":[]},"discord-sessions-prd-2-139",{"micros":55830,"calls":["start_session",{"micros":42895,"calls":["discord-api-dc69f8484-vv6j9",{"micros":37359,"calls":["get_user",{"micros":7310},"get_guilds",{"micros":3581},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":821},"authorized_ip_coro",{"micros":10}]}]},"starting_guild_connect",{"micros":69,"calls":[]},"presence_started",{"micros":361,"calls":[]},"guilds_started",{"micros":176,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":12294,"calls":[]},"connect_finished",{"micros":12301,"calls":[]},"build_ready",{"micros":24,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":0,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}]']
[11:48:46] [DEBUG] Dispatching Event: websocket_ready
[11:48:47] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[11:48:47] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[11:48:47] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[11:48:47] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[11:48:47] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[11:48:47] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[11:48:47] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[11:48:47] [DEBUG] 0 is already up-to-date with 15 commands.
[11:48:47] [DEBUG] Sync of 1 scopes took 0.8022664000018267 seconds
[11:48:47] [DEBUG] Dispatching Event: startup
[11:48:47] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1702961327607, "activities": [{"name": "/help for helpful", "type": 0}], "status": "idle", "afk": false}}
[11:48:54] [ERROR] Jurigged encountered an error
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\jurigged\codetools.py", line 493, in _process_child_correspondence
    orig.apply_correspondence(
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\jurigged\codetools.py", line 704, in apply_correspondence
    new_obj = self.reevaluate(corr.new.node, glb)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\jurigged\codetools.py", line 801, in reevaluate
    conform(self.get_object(), new_obj)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\codefind\registry.py", line 118, in conform
    self.conform(obj1, obj2.__code__)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\codefind\registry.py", line 123, in conform
    fv1 = obj1.__code__.co_freevars
          ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute '__code__'
[11:48:54] [DEBUG] Update __main__:1
[11:49:06] [DEBUG] Sending heartbeat every 41.25 seconds
[11:49:06] [DEBUG] Sending data to websocket: {"op": 1, "d": 6}
[11:49:06] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[11:49:07] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[11:49:30] [DEBUG] Dispatching Event: disconnect
[11:49:30] [DEBUG] Stopping the bot.
[11:49:30] [INFO] Shard ID 0 | Stopping Shard
